test/integration/protocol_integration_test.cc

```diff
@@ -339,30 +339,33 @@ TEST_P(DownstreamProtocolIntegrationTest, RetryPriority) {
   auto retry_policy = host.mutable_routes(0)->mutable_route()->mutable_retry_policy();
   retry_policy->mutable_retry_priority()->set_name(factory.name());
   config_helper_.addVirtualHost(host);
-
-  // Use load assignments instead of static hosts. Necessary in order to use priorities.
+  // We want to work with a cluster with two hosts.
   config_helper_.addConfigModifier([](envoy::config::bootstrap::v3alpha::Bootstrap& bootstrap) {
-    auto cluster = bootstrap.mutable_static_resources()->mutable_clusters(0);
-    auto load_assignment = cluster->mutable_load_assignment();
+    auto* cluster = bootstrap.mutable_static_resources()->mutable_clusters(0);
+    const std::string host_address = cluster->load_assignment()
+                                         .endpoints(0)
+                                         .lb_endpoints(0)
+                                         .endpoint()
```

Comment:
Should there by a test helper to get the first endpoint in this common scenario?

Reply:
Not sure, there are only a few occurrences of this, and by time we end up wrapping the load assignment with some helper the code is almost as long. To some extent, it's an anti-pattern relying on the first host as a template for copying, I rather that the test framework had some `createReferenceHost()` method.. actually, I think I'll switch to that instead! :)